src/main/resources/graphql/person.graphqls

```diff
@@ -0,0 +1,33 @@
+type Document {
+  id: ID!
+  title: String!
+  abstractText: String
+  authorList: [String]
+  doi: String
+  issn: String
+  pageStart: String
+  pageEnd: String
+  volume: String
+  issue: String
+}
+
+type Person {
+  id: ID!
+  name: String!
+  preferredTitle: String
+  primaryEmail: String
+  additionalEmails: [String]
+  overview: String
+  image: String
+  thumbnail: String
+  documents: [Document]
+}
+
+input Paging {
+  page: Int
+  pageSize: Int
+}
+
+type Query {
+    people(search: String, paging: Paging!): [Person]!
+}
```

Comment:
hi @nymbyl since this is called `person.graphqls`, will there be separate files for other top-level entities?

Reply:
I did copy what Jim had done, so I don't know if it needed to be called that specifically.  I guess my first preference would be a single schema file.  But maybe that would be too big.

Reply:
I tend to agree with having one larger file until we out grow it.